codespell_lib/data/dictionary.txt

```diff
@@ -13401,6 +13431,7 @@ theres->there's
 therfore->therefore
 theri->their, there,
 thes->this, these,
+theses->these, thesis,
```

Comment:
`theses` is the plural of `thesis` so should be allowed as a correction

Reply:
Don't we normally remove them if the incorrect word is actually valid?

Reply:
Sometimes yes, sometimes no. Depends on the likely balance of false alarms and misses

Reply:
That should probably be another bit in the bitmask:
https://github.com/codespell-project/codespell/issues/410#issuecomment-430253787

Reply:
Looks the same as the other cases we were talking about there, so I still only see a need for 0/1 so far

Reply:
Ah, maybe I was thinking of "disabled due to" instead.